tool/generate_spdx_license_list.rb

```diff
@@ -7,21 +7,29 @@
 def generate_spdx_license_list(dest = "lib/rubygems/util/licenses.rb")
```

Comment:
This method is gnarly. How often do we need to run this? I want to avoid paying off this technical debt if this is almost never run, but I did want to file a formal complaint about the RUBY heredocs in the middle.

Reply:
Hey, this is a massive improvement over when there was no script to be ran every two years at all

Reply:
What do you mean @martinemde? Migrate this into erb template to render instead of inline heredoc?

Reply:
I don't think it's a big deal right now, but if this was a piece of code we had to regularly run and maintain, I would spend time trying to make it easier to read.

If I had to refactor it, just off the top of my head I'd avoid any interpolation at all and just write the data to files, then read them in for use in the script. Still, I think there's probably more important work than this, I was just noting how hard that code was to read the first time through.